src/main/java/org/sunbird/workflow/service/impl/WorkflowServiceImpl.java

```diff
@@ -1363,8 +1363,44 @@ public Response getUserProfileApprovalRequest(String rootOrg, String org, Search
 			Pageable pageable = getPageReqForApplicationSearch(criteria);
 			List<String> applicationIds = criteria.getApplicationIds();
 			long totalRequestCount = 0;
-
-			if (CollectionUtils.isEmpty(applicationIds)) {
+			if (StringUtil.isNotBlank(criteria.getQuery()) && criteria.getServiceName().equals(Constants.PROFILE_SERVICE_NAME)) {
+				if (StringUtil.isBlank(rootOrgId) && (criteria.getRequestType().equals("GROUP_CHANGE") || criteria.getRequestType().equals("DESIGNATION_CHANGE"))) {
```

Comment:
Can you check which API is getting called when you open user profile page ??
there also we search for workflow service request --